linear_api/managers/issue_manager.py

```
@@ -738,14 +742,125 @@ def get_comments(self, issue_id: str) -> List[Comment]:
             ["issue", "comments", "nodes"],
-            Comment  # Pass the model class for automatic conversion
+            Comment,  # Pass the model class for automatic conversion
         )
 
         # Cache the result
         self._cache_set("comments_by_issue", issue_id, comments)
 
         return comments
 
+    def create_comment(
+        self, issue_id: str, body: str, username: str, parent_id: str = None
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)